provisioning/transport/mqtt/transport_mqtt.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26730.12
+VisualStudioVersion = 15.0.27004.2008
```

Comment:
when is it necessary to update this version?

Reply:
VS got an update and for some reason it likes to update the SLNs that it touched. 
I'll just go with it unless you see a problem opening SLNs on systems that are not up to date.